### PR TITLE
Improve docs for `std::env::args()`

### PR DESCRIPTION
**Pull request by camelid**
@rustbot modify labels: T-doc

### CODE REVIEW

File: library/std/src/env.rs [hunk 1]
```diff
@@ -695,21 +695,21 @@ pub struct ArgsOs {
     inner: sys::args::Args,
 }
 
-/// Returns the arguments which this program was started with (normally passed
```
Comment (pickfire) [inline library/std/src/env.rs:698]: I think which is correct. Is it not?

File: library/std/src/env.rs [hunk 2]
```diff
@@ -695,21 +695,21 @@ pub struct ArgsOs {
...
+/// Returns the arguments that this program was started with (normally passed
 /// via the command line).
 ///
 /// The first element is traditionally the path of the executable, but it can be
 /// set to arbitrary text, and may not even exist. This means this property should
 /// not be relied upon for security purposes.
 ///
-/// On Unix systems shell usually expands unquoted arguments with glob patterns
+/// On Unix systems the shell usually expands unquoted arguments with glob patterns
```
Comment (pickfire) [inline library/std/src/env.rs:705]:
```suggestion
/// On Unix systems, the shell usually expands unquoted arguments with glob patterns
```

File: library/std/src/env.rs (hunk 1 above)
Comment (camelid) [reply library/std/src/env.rs:698]: I believe “that” is correct because this is a defining clause.

File: library/std/src/env.rs (hunk 2 above)
Comment (camelid) [reply library/std/src/env.rs:705]: My understanding is that commas are usually recommended after four or more words of an introductory phrase and it sounds smoother to me without the comma.

**Review by pickfire — APPROVED** [review]
Looks good to me.

**Pull request merged**